packages/@ama-sdk/schematics/schematics/typescript/core/helpers/copy-referenced-files.ts

```diff
@@ -0,0 +1,96 @@
+import { existsSync } from 'node:fs';
+import { copyFile, mkdir, readFile, rm } from 'node:fs/promises';
+import { dirname, join, normalize, posix, relative, resolve } from 'node:path';
+
+const refMatcher = /\B['"]?[$]ref['"]?\s*:\s*([^#\n]+)/g;
```

Comment:
```suggestion
const refMatcher = /\B['"]?[$]ref['"]?\s*:[\n\s|]*([^#\n]+)/g;
```

Reply:
![image](https://github.com/user-attachments/assets/21e8cb24-f6a1-411d-831d-79974a7f3325)
Not sure this is a valid use-case